apps/meteor/server/models/raw/LivechatRooms.js

```diff
@@ -1,8 +1,10 @@
 import { escapeRegExp } from '@rocket.chat/string-helpers';
+import { Settings } from '@rocket.chat/models';
 
 import { BaseRaw } from './BaseRaw';
 import { getValue } from '../../../app/settings/server/raw';
 import { readSecondaryPreferred } from '../../database/readSecondaryPreferred';
+import { trim } from '../../../lib/utils/stringUtils';
```

Comment:
this is causing problem since it depends on `dompurify` to provide a `stripTags` function, which you don't even use.

I'd suggest to use native `trim` ;)